client/dist/styles.css

```diff
@@ -57,7 +62,7 @@ body {
   flex-direction: row;
   border: solid 2px black;
   height: 442px;
-  width: 100%;
+  width: 99.54%;
```

Comment:
This can easily be resolved with:

```
*, *:before, *:after {
  box-sizing: border-box;
}
```

At the top of your `index.css` file.  Try it out, see if it ruins the styles you've already built. If ruined, disregard, if not, use this as it's better :)